kubernetes/examples/mnist/deployment.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ts-def
+  labels:
+    app: ts-def
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: ts-def
+  template:
+    metadata:
+      labels:
+        app: ts-def
+    spec:
+      volumes:
+        - name: model-store
+          hostPath:
+            path: /host/model_store
+      containers:
+        - name: torchserve
+          image: pytorch/torchserve:latest-cpu
```

Comment:
We also ship a kserve image, would that be more appropriate to use? Look at Dockerfile here https://github.com/pytorch/serve/tree/master/kubernetes